engines/scumm/string.cpp

```diff
@@ -541,6 +541,8 @@ bool ScummEngine::newLine() {
 	} else if (_isRTL) {
 		if (_game.id == GID_MANIAC || ((_game.id == GID_MONKEY || _game.id == GID_MONKEY2) && _charset->getCurID() == 4)) {
 			_nextLeft = _screenWidth - _charset->getStringWidth(0, _charsetBuffer + _charsetBufPos) - _nextLeft;
+		} else if (_game.id == GID_MONKEY2 && _charset->getCurID() == 5) {
+			_nextLeft += _screenWidth - 210 - _charset->getStringWidth(0, _charsetBuffer + _charsetBufPos);
```

Comment:
What's this 210 magic number?

Reply:
I'm not actually sure because it was looked accurate on first try (measured 4 pixels diff from border).
my guess it's that is the total width excluding the card + margin.